src/crypto-browser.js

```diff
@@ -6,14 +8,13 @@ const nodeify = require('nodeify')
 const webCrypto = getWebCrypto()
 
 function getWebCrypto () {
```

Comment:
Yeah if you could do this please. I think I forgot to clean this up when the second one was introduced.